### PR TITLE
Fix NCMenu in landscape

### LINKED ISSUE #1810: NCMenu not displayed correctly after device rotation
### Steps to reproduce
1. start the app in portrait orientation
2. tap on "+"
3. rotate the device to landscape orientation
4. try to scroll through the menu

### Expected behaviour
The menu should be updated to display all actions as if it were triggered in landscape

### Actual behaviour
The menu is not displayed in total and it's not possible to scroll through the selection menu

### Environment data
Device: iPad

### PR DESCRIPTION
**Pull request by thisIsTheFoxe**
Fixes #1810 
- Update Swift Package versions 
  - Update Floating Panel ~> v2

### CODE REVIEW

**Comment by marinofaggiana** [comment]
Hi @thisIsTheFoxe you write

> Fixes issues with the NCMenu in landscape orientation

where are the issue ? (good sync version)

**Review by marinofaggiana — CHANGES_REQUESTED** [review]
Do not works, job incomplete

**Comment by thisIsTheFoxe** [comment]
Sorry, @marinofaggiana I had some trouble building for iPad simulator so that made it a little tricker.. could you take another look?

**Comment by marinofaggiana** [comment]
@thisIsTheFoxe with a touch in background (NO menu) the menu don't close

**Review by marinofaggiana — CHANGES_REQUESTED** [review]
Menu do dot close with touch outside

File: iOSClient/Menu/NCMenu.swift [hunk 1]
```diff
@@ -91,13 +91,6 @@ class NCMenu: UITableViewController {
     override func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
         return 60
     }
-
-    // MARK: - Accessibility
-
-    open override func accessibilityPerformEscape() -> Bool {
-        dismiss(animated: true)
-        return true
-    }
```
Comment (thisIsTheFoxe) [inline iOSClient/Menu/NCMenu.swift:92]: @marinofaggiana can you confirm that this was only used for dismissing when tapping outside the menu?

Comment (marinofaggiana) [reply iOSClient/Menu/NCMenu.swift:92]: Sorry, I don't have write this code, you have to test it by yourself

Comment (thisIsTheFoxe) [reply iOSClient/Menu/NCMenu.swift:92]: I always test manually. But I just wasn't sure if it's needed for anything else. I suppose it's ok then.

**Comment by thisIsTheFoxe** [comment]
@marinofaggiana can you please have another look at this?

**Comment by marinofaggiana** [comment]
PLEASE TEST:

https://user-images.githubusercontent.com/8616947/146160205-5b0f973a-a988-453d-839a-92144932743b.mov

**Review by marinofaggiana — CHANGES_REQUESTED** [review]
Don't works

**Comment by thisIsTheFoxe** [comment]
@marinofaggiana looks like bug where the UIScreen height isn't updated before calling `layoutFor size`. Should be fixed now.

**Comment by marinofaggiana** [comment]
@thisIsTheFoxe no problem but why didn't you try it ?  I have found this issue after 2 seconds. this PR is open from 8 day ago ... please open only one PR work on until works correctly after merge and after open another PR.

**Comment by thisIsTheFoxe** [comment]
@marinofaggiana As per this PR and its linked issue I was focused on the UI on the iPad, which worked fine. I tested it with iPhone as well, but admittedly not for the previous commit. That's why it good that we do PRs so that issues like that can be noticed before it goes 'live'.
Thanks for your previous review. As said, the issue should now be fixed. Please have another look..

**Comment by marinofaggiana** [comment]
There is a conflict, @thisIsTheFoxe before open a PR you can tested in your branch with all type of emulator (iPhone/iPad)... creating long discussions takes me too long time, I can do a quick test but it is always assumed that whoever did it has always tried it, so also for the others PR or even worse for the communication library, you have to be 100% sure before creating a PR.

**Review by marinofaggiana — APPROVED** [review]

**Pull request merged**